ci/tasks/setup-infrastructure.sh

```diff
@@ -51,5 +52,9 @@ gcloud -q compute http-health-checks create ${google_backend_service}
 gcloud -q compute backend-services create ${google_backend_service} --http-health-check ${google_backend_service} --port-name "http" --timeout "30"
 gcloud -q compute backend-services add-backend ${google_backend_service} --instance-group ${google_backend_service} --zone ${google_zone} --balancing-mode "UTILIZATION" --capacity-scaler "1" --max-utilization "0.8"
 
-
+# Region Backend service
+gcloud -q compute instance-groups unmanaged create ${google_region_backend_service} --zone ${google_zone}
+gcloud -q compute health-checks create tcp ${google_region_backend_service}
+gcloud -q compute backend-services create ${google_region_backend_service} --region ${google_region} --health-checks ${google_region_backend_service} --protocol "TCP" --load_balancing_scheme "INTERNAL" --timeout "30"
+gcloud -q compute backend-services add-backend ${google_region_backend_service} --instance-group ${google_region_backend_service} --zone ${google_zone} --region ${google_region}
```

Comment:
@danawillow I'm seeing this error in CI:

```
Instance group must have a network to be attached to a backend service. Add an instance to give the instance group a network.
```

AFAICT there's not a way to specify a network for the IG (or anywhere in the backend service). The global backend service doesn't have this problem. I'm guessing it's specific to regional? If that's the case, I suppose the ugly fix is to create an instance and put it in the IG.

I pushed a few small fixes to get to this point, so you'll want to rebase on `develop` before digging in.